repos.d/apkfission.yaml

```diff
@@ -0,0 +1,40 @@
+###########################################################################
+# APK Fission packages
+#
+# https://apkfission.net/install.html
+###########################################################################
+
+{% macro apkfission(version, subrepos=['main','nonfree'], archs=['aarch64','armv7','pmmx','ppc','ppc64','x86_64']) %}
+- name: apkfission_adelie_{{version|replace('.', '_')}}
+  type: repository
+  desc: APK Fission for Adélie {{version|title}}
+  statsgroup: APKFission
+  family: apkfission
+  default_maintainer: fallback-mnt-apkfission@repology
+  sources:
+    {% for subrepo in subrepos %}
+    {% for arch in archs %}
+    - name: {{subrepo}}/{{arch}}
+      fetcher:
+        class: TarFetcher
+        url: 'http://distfiles.apkfission.net/adelie/{{version}}/{{subrepo}}/{{arch}}/APKINDEX.tar.gz'
+      parser:
+        class: ApkIndexParser
+      subrepo: {{subrepo}}/{{arch}}
+    {% endfor %}
+    {% endfor %}
+  repolinks:
+    - desc: APK Fission home
+      url: https://apkfission.net/
+  packagelinks:
+    - type: PACKAGE_SOURCES
+      url: 'https://git.adelielinux.org/apkfission/packages/-/tree/master/{subrepo}/{srcname}'
```

Comment:
This may be confusing, but it takes subrepo set in `sources` to `{{subrepo}}/{{arch}}`, so it'll include arch which you don't want. You may move `packagelinks` block to `source` and use Jinja templates there, e.g. 'https://git.adelielinux.org/apkfission/packages/-/tree/master/{{subrepo}}/{srcname}'